templates/openstack/compare/contact-us.html

```diff
@@ -12,213 +12,6 @@
 
 {% block content %}
 
-  <section class="p-strip is-shallow u-no-padding--bottom">
-    <div class="p-section">
-      <div class="row">
-        <div class="col-start-large-4 col-8">
-          <h1>Contact the Canonical team</h1>
-          <p class="p-lead">Contact us to learn more about Canonical OpenStack and how it compares to other alternatives.</p>
-        </div>
-      </div>
-    </div>
-  </section>
-
-  <section class="p-section">
-
-    <form action="https://ubuntu.com/marketo/submit"
-          method="post"
-          id="mktoForm_4271">
```

Comment:
In the form-data.json file, the marketo id is 6464 instead of 4271
https://github.com/canonical/canonical.com/pull/1975/files#diff-ffe5c6d184c6d486b5e2419e776c4b11478f28892bf3c8ab51da224339b857deR46

Reply:
Right.
Thanks for the comment